client/landing/gutenboarding/style.scss

```diff
@@ -27,10 +27,15 @@ $admin-sidebar-width-collapsed: 0;
 @import '~@wordpress/block-editor/src/style.scss';
 @import '~@wordpress/format-library/src/style.scss';
 
-.gutenboard__edit-post-layout-content {
+.gutenboarding__content {
 	position: static;
 }
 
+.gutenboarding__content-editor {
+	// overrides .edit-post-visual-editor
+	padding-top: $header-height;
```

Comment:
Coming from here:

https://github.com/WordPress/gutenberg/blob/70eef3a076787043e8d960d70cae81724e69adcf/packages/base-styles/_variables.scss#L42

Value of `60px` currently.